scripts/mc_gps.py

```diff
@@ -4,7 +4,7 @@
 # Licensed under the 2-clause BSD license.
 
 """
-Prints Time info for given date/time
+Prints Time info for given date/time.  This is being deprecated for mc_clock.py
```

Comment:
While this information is useful for developers, the script itself should probably issue a warning when someone tries to use it. Telling them that it is being deprecated, what to use in its stead, and when they can expect this script to disappear.